src/trace_local.rs

```diff
@@ -118,18 +139,18 @@ impl Drop for LocalTraceGuard {
     fn drop(&mut self) {
         let tl = unsafe { &mut *self.trace_local };
 
-        // fill the elapsed cycles of the first span
-        tl.spans[self.span_start_index].elapsed_cycles =
-            minstant::now().saturating_sub(tl.spans[self.span_start_index].begin_cycles);
+        // fill the elapsed cycles of the first span (except the leading span)
```

Comment:
Any explanation about why created a new concept `LeadingSpan` and change logic here?

Reply:
A leading span is one of `Root`, `Spawning`, and `Scheduling`, which represents how a tracing context creates.

The elapsed time of a leading span means how long it waits from task creation to starting execution. 

A `State` span means the execution is starting.

Reply:
> A leading span is one of `Root`, `Spawning`, and `Scheduling`, which represents how a tracing context creates.
> 
> The elapsed time of a leading span means how long it waits from creating the task to starting execution.
> 
> A `Settle` span means the execution is starting.

Maybe this response can add to the comments in L142